.github/workflows/release-ci.yml

```diff
@@ -0,0 +1,87 @@
+name: npm - release to dev/integration
+run-name: npm - release from ${{ github.ref_name }} to dev/integration
+
+on:
+  push:
+    branches:
+      - main
+      - 'integration/*'
+      - '*.*.x'
+    paths-ignore:
+      - '*.md'
+      - '.devcontainer/**'
+      - '.vscode/**'
+      - '.github/ISSUE_TEMPLATE/**'
+      - 'LICENSE'
+      - '.github/CODEOWNERS'
+      - '.buildkite/**'
+      - '.dockerignore'
+      - '*.bench.ts'
+      - 'graphs/**'
+  workflow_dispatch:
+
+env:
+  # To hide "Update available 0.0.0 -> x.x.x"
+  - PRISMA_HIDE_UPDATE_MESSAGE="true"
+
+jobs:
+  release:
+    timeout-minutes: 60
+    environment: 'npm-ci'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install & build
+        uses: ./.github/actions/setup
+        with:
+          node-version: 16
+
+      - name: Publish all packages to npm
+        run: pnpm run publish-all
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          BUILDKITE: true
```

Comment:
It's weird that it's still have `BUILDKITE`, `BUILKITE_BRANCH` and `BUILDKITE_TAGS` vars. Could we rename them, say, to `RELEASE_*`?

Reply:
Yes indeed, I didn't want to change too many things right away, the PR is already "big"

Reply:
Wait for cleanup PR coming later 😉